its/plugin/projects/externalreport/scalastyle/scapegoat-scalastyle.xml

```diff
@@ -0,0 +1,13 @@
+<checkstyle version="5.0" generatedBy="scapegoat">
```

Comment:
The name of the file is strange: scapegoat-scalastyle.xml. Is it scapegoat or is it scalastyle?

Reply:
It's the default name used by scapegoat to save a report using scalastyle format.